### PR TITLE
Fix collections/series meta tags

### LINKED ISSUE #13509: Wrong twitter/facebook info for series pages
I just noticed that the "og" meta and "twitter" meta are not correctly set for the series page. They are kept generic.

Example of serie: https://dev.to/afif/series/12016 that I tried to share on Twitter: https://twitter.com/ChallengesCss/status/1386328494498533377

### PR DESCRIPTION
**Pull request by jeferson-sb**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Update series article meta tags content (open graph, twitter, etc) to properly match series information

## Related Tickets & Documents

Closes #13509 

## Added tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

**Comment by juliannatetreault** [comment]
Thank you so much for working on this bugfix, @jeferson-sb -- it's looking great so far!!

That said though, I checked out your work and ran into a `schema`-related issue locally (we made some recent changes to some database columns the other day and it looks like the database differences I'm seeing might be related to that!). If you wouldn't mind, could you please sync your branch up with the latest version of `main`? Once you have synced your branch with `main`, I will continue QAing your work! Thank you so much in advance. 😄

File: app/views/collections/_meta.html.erb [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-<meta property="og:description" content="<%= community_name %> is great!" />
+<meta property="og:description" content="<%= description_string %>" />
 <meta property="og:site_name" content="<%= community_name %>" />
 
 <meta name="twitter:card" content="summary_large_image">
 <meta name="twitter:site" content="@<%= SiteConfig.social_media_handles["twitter"] %>">
-<meta name="twitter:title" content="About <%= community_name %>">
-<meta name="twitter:description" content="<%= community_name %> is great!">
+<meta name="twitter:title" content="<%= title_string %>">
+<meta name="twitter:creator" content="@<%= @user.twitter_username %>">
```
Comment (rhymes) [inline app/views/collections/_meta.html.erb:17]: This should be optional, `.twitter_username` can be empty as a user could not have a Twitter username

**Review by rhymes — COMMENTED** [review]
Thanks @jeferson-sb!

There's a conflict to be solved and I think we should add a simple test as well, you can use https://github.com/forem/forem/blob/main/spec/requests/collections_spec.rb

Let us know if you need any help!

File: app/views/collections/_meta.html.erb (hunk 1 above)
Comment (jeferson-sb) [reply app/views/collections/_meta.html.erb]: Fixed, thanks for the help!

**Review by rhymes — APPROVED** [review]
Thanks!

File: app/views/collections/_meta.html.erb [hunk 2]
```diff
@@ -1,17 +1,18 @@
 <% title title_string %>
-<link rel="canonical" href="<%= app_url("/about") %>" />
-<meta name="description" content="<%= title_string %> on <%= community_name %>">
+
+<link rel="canonical" href="<%= app_url(path) %>" />
```
Comment (nickytonline) [inline app/views/collections/_meta.html.erb:3]: @rhymes, quick question. Does a canonical URL make sense in the context of a series? As far as I know we don't have a mechanism to specify a canonical URL for a series, only a post.

Comment (rhymes) [reply app/views/collections/_meta.html.erb:3]: Good point, you're right, I don't think a Forem series can have a duplicate place on the web, it's not like a post that can come from multiple places.

@jeferson-sb would it be okay for you to remove the canonical URL from the tags?

**Review by juliannatetreault — APPROVED** [review]
This looks great, @jeferson-sb! Thank you so much!

**Pull request merged**